app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
```

Comment:
Do you need this conditions in this file?

Reply:
I am not sure about that, but... I think that we need in this, because this code works with "default" Devise implementation. In future, when i will scales my application, i can edit that and. I thinks, it necessary